util/rom_chip_info.py

```diff
@@ -16,15 +15,11 @@ def generate_chip_info_c_source(scm_revision: int) -> str:
     Args:
         scm_revision: SHA1 sum identifying the current SCM revision.
     """
-
-    SHA1_DIGEST_SIZE_BITS = hashlib.sha1().digest_size * 8
-    assert scm_revision.bit_length() <= SHA1_DIGEST_SIZE_BITS
-    assert scm_revision >= 0
-
-    # Truncate the SCM revision to the top N bits.
-    SCM_REVISION_SIZE_BITS = 64
-    shift_amount = SHA1_DIGEST_SIZE_BITS - SCM_REVISION_SIZE_BITS
-    scm_revision >>= shift_amount
+    SHA1_BYTE_CNT = 20
+    assert 0 < scm_revision < 2**(SHA1_BYTE_CNT * 8)
+    scm_rev_byte_be = scm_revision.to_bytes(SHA1_BYTE_CNT, "big")
```

Comment:
Optional nit: it might be nice to have a comment that highlights the fact that we're truncating a 160-bit commit hash to 64 bits.